tensorflow_lite_support/cc/test/task/processor/BUILD

```diff
@@ -0,0 +1,34 @@
+load(
+    "@org_tensorflow//tensorflow/lite/core/shims:cc_library_with_tflite.bzl",
+    "cc_test_with_tflite",
+)
+
+package(
+    default_visibility = [
+        "//tensorflow_lite_support:users",
+    ],
+    licenses = ["notice"],  # Apache 2.0
+)
+
+cc_test_with_tflite(
+    name = "image_preprocessor_test",
+    srcs = ["image_preprocessor_test.cc"],
+    data = [
+        "//tensorflow_lite_support/cc/test/testdata/task/vision:test_images",
+        "//tensorflow_lite_support/cc/test/testdata/task/vision:test_models",
+    ],
+    deps = [
+        "@org_tensorflow//tensorflow/lite/core/shims:cc_shims_test_util",
+        "//tensorflow_lite_support/cc/task/core:task_api_factory",
```

Comment:
task_api_factory can be removed, as well as its header file in image_preprocessor_test.cc.